### DISCUSSION TITLE
No cell outline in "empty" cells

### DISCUSSION THREAD

**Discussion by sofia-bg** [UNC-Project-Open-AAC/OS-DPI#170] [Ideas]
Beyond putting in gaps between grids, is there a way in a single large grid for a cell without content have it's outline erased? So instead of the empty light gray outlines below, it would all just be empty space, perhaps even the same color as the stack background color? 

<img width="758" alt="Screen Shot 2023-03-01 at 9 38 31 AM" src="https://user-images.githubusercontent.com/92817124/222171926-d8426dd6-02ca-4774-859c-ce86c9c5ebd3.png">

**Comment by gbishop** [comment]
Yes, this and much more is possible using the Customize component.

The following CSS will do what (I think) you want:

.grid button:disabled {
  border: none;
}

**Reply by sofia-bg** [reply]
Thanks, I'll give it a try next time I have a chance to play!

**Reply by sofia-bg** [reply]
@gbishop also re: color - is there a way that we could declare that words with #partofspeech = 'verb' (for example) are a different color than other words in same grid? Right now could make separate grids with separate colors and populate by a column #partofspeech (or whatever else is of interest), but curious if we could do within programmer interface without making designer tab super long with stacks etc.

**Comment by gbishop** [comment]
Yes, Customize is crazy powerful.

.grid button[#partofspeech="verb"] {
  color: red;
}

That should make verbs show up in red letters. Change color to
background-color and you can set the background...

**Reply by sofia-bg** [reply]
ah ok seeing a pattern here...!

**Comment by gbishop** [comment]
Yes. I decorate the buttons with attributes that have all the field values
associated with it. Then I hack the CSS you are writing so that

button[#partofspeech="verb"] becomes the legal CSS
button[data-partofspeech="verb"]

You have to learn the sometimes obscure CSS properties but they are pretty
easy to Google. For example: https://www.w3schools.com/css/css3_buttons.asp
